parquet-column/src/main/java/org/apache/parquet/example/data/simple/SimpleGroup.java

```diff
@@ -44,10 +44,10 @@ public SimpleGroup(GroupType schema) {
 
   @Override
   public String toString() {
-    return toString("");
+    return toString(new StringBuilder()).toString();
   }
 
-  public String toString(String indent) {
+  public StringBuilder toString(StringBuilder indent) {
```

Comment:
We shall not modify existing public methods as they are breaking changes. That's why I've mentioned creating a new private method that would be invoked by the existing ones.
Let me put an example about the idea I wanted to describe:
```java
public String toString(String indent) {
  StringBuilder builder = new StringBuilder();
  appendToString(builder, indent);
  return builder.toString();
}

private void appendToString(StringBuilder builder, String indent) {
  // ...
        builder.append(indent).append(name);
        if (value == null) {
          builder.append(": NULL\N");
        } else if (value instanceof Group) {
          builder.append('\n');
          ((SimpleGroup) value).appendToString(builder, indent + "  ");
        } else {
          builder.append(": ").append(value).append('\n');
  // ...
}
```

Reply:
Ok make sense. have done the required changes.